### DISCUSSION TITLE
Possible completion options listed is differs from what <app> --help displays

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [marlonrichert/zsh-autocomplete#615] [Q&A]
This happens with quite a few apps but I've ignored it till now. For e.g. when using the CLI for `Fly.io` at the prompt I type:
`> fly ` i.e, fly followed by space and I get a list of completion options like below:

<img width="867" alt="Screenshot 2023-07-11 at 09 42 53" src="https://github.com/marlonrichert/zsh-autocomplete/assets/219075/264a7363-343f-4c67-b4d7-088411ee2584">

However there are optons that I don't see in the list provided. Indeed the list I see is quite different from what I get from 
`> fly --help` blelow:
`...`
<img width="895" alt="Screenshot 2023-07-11 at 09 47 22" src="https://github.com/marlonrichert/zsh-autocomplete/assets/219075/6ce0651e-746e-4b10-9e26-8ef874a409a3">
`...`

I installed the app `flyctl` using homebrew. Could someone point out what I'm doing wrong?

**Comment by marlonrichert** [comment]
Completions for most commands are not generated from the `--help` text, but maintained manually. In the case of `fly`, they are maintained in [`zsh-users/zsh-completions`](https://github.com/zsh-users/zsh-completions). Please file a bug over there.
